.github/workflows/fix_permissions.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Overall, need to add comments & descriptions in this file to understand what & why we're doing so it's clear to people like me without needing to have access to ask questions from experts like you.